### ISSUE TITLE
[Accessibility] User Report: New Site Address Input Not Found in Screen Reader

### ISSUE THREAD

**Issue by scosgro** [Automattic/wp-calypso#30071]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/start/domains
2. User reported their screen reader, JAWS 2019, was not able to find the form at the top to search for a new site URL.
3. It found the Filters button, but not the search form.

#### What I expected
For the user to be able to immediately find the search form, and search for and register a site address.

#### What happened instead
They were stuck, unable to find the search form. Their screen reader, JAWS 2019, could not locate it. 

#### Context / Source

We'd also love to know how you found the bug:  #user-report

**Comment by scosgro** [comment]
> @scosgro do we know which web browser is being used in conjunction with JAWS?

I don't. I've reached out to the user asking for more information (web browser/OS). 
#1707665-zen

**Comment by lcollette** [comment]
I can't repro this bug. Closing it out for now.
